src/frontend/src/lib/types/i18n.d.ts

```diff
@@ -72,6 +72,11 @@ interface I18nAuth {
 	};
 }
 
+interface I18nDapps {
```

Comment:
I imagine that this is because this strings where not auto-generated with the script in another PR

Reply:
No that's because on `main` it's already like that - i.e. `I18nDapps` is on `main`.

Reply:
Ah you mean the order?

Reply:
Re-run the script, it's correct, the order changed.

Reply:
I guess it's because I added "signer" at the end of the script.

Works for you?

Reply:
I think it was nothing to be done I imagine: indeed it was like that on main because I suppose it was added manually in some PR before this one. No worries, than kyou for checking